
//try catch 
/*
int x=10;
int y=0; // Run the program and see what's happening
Console.WriteLine(x/y);
*/
// // to avoid this kind of issue, we block our code in Try Catch 

// int x1=10, y1=0;
// try
// {
//   Console.WriteLine(x1/y1);
// }
// catch(Exception ex)
//  {
//    Console.WriteLine("Cuase of problem is :" + ex);
// }
// finally //  you can add this Finally block if your code has to run certain code
// { // whether Try block  run or Catch block run
//   Console.WriteLine("Done");
// }

// if(y1==0)
// {
//  throw new InvalidOperationException("Cannot divid by Zero");
// }
// else
// {
//    Console.WriteLine(x1/y1);  
// }